### PR TITLE
#58425280 Add feature for drivers to view all rides they have given

### PR DESCRIPTION
**Pull request by meshack-mbuvi**
**What does this PR do?**
This pull request creates _driver.html_ showing a table of rides driver has given.

**Description of Task to be completed?**
At minimum, the driver.html page should have the following:
-Table with records for rides driver has given
- Option to add a new ride which links to _new_ride.html_
- Links for driver to delete or edit records.

**How should this be manually tested?**
- Clone this repository
- In your terminal, cd ride-my-way
- `git checkout ft-view-rides-given-158425280`
-  Open driver.html using your favorite browser.

**What are the relevant pivotal tracker stories?**
[[#158425280]](https://www.pivotaltracker.com/story/show/158425280)

**Screenshot**
driver_rides

### CODE REVIEW

**Comment by Ogollah** [comment]
Good work there. keep it up!

File: ui/driver.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<!--Make width of page follow the screen-width of the device-->
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+	<meta charset="UTF-8">
+
```
Comment (rethes) [inline ui/driver.html]: Work is okay, just avoid the unnecessary white spaces

**Comment by TheSteelGuy** [comment]
Nice one

**Comment by LWanjiru** [comment]
Great UI! Quick question, will the driver be editing information on the rides they have given? (asking because I spotted an `Edit` icon in your `Action` column.
Looks good to merge.

**Comment by meshack-mbuvi** [comment]
I was of the idea that driver may want to add more notes about the ride.

**Pull request merged**